packages/react-ui-components/src/_lib/withValidationResultRendering.js

```diff
@@ -0,0 +1,33 @@
+import React, {PureComponent} from 'react';
+import PropTypes from 'prop-types';
+
+function withValidationResultsRendering(WrappedComponent) {
+    class WithValidationResultsRendering extends PureComponent {
+
+        static propTypes = {
+            /**
+             * An array of error messages
+             */
+            validationErrors: PropTypes.array
+        }
+
+        render() {
+            const {validationErrors, ...rest} = this.props;
+
+            const renderedErros = validationErrors && validationErrors.length > 0 && validationErrors.map((validationError, key) => {
+                return <div key={key}>{validationError}</div>;
```

Comment:
shouldn't the validation error be rendered as a tooltip; or did I misunderstand your change? :)

Reply:
Yeah, we did plan that but doesn't work out really good so far. Hard to get it really sticky, overwriting styles is also not an easy task right now. Also i had some problem with a setState that is triggered inside the ReactTolltip Component and moun/unounting if they get rerendered. So just for now we thought might be better to get in v1 the tooltip as a component in and implement it in other parts of the ui in a follow up. This change also has not yet all tooltips set, just a few for a demo right now

Reply:
OK :)